hooks/build

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+docker build \
```

Comment:
Unncessary hook, right?
If you really need it, add it to the Makefile.
You can update it with:
```IMAGE_NAME=base
$(IMAGE_NAME):
           	cd $(MAIN_REPO) && docker build ....
``` and call the makefile with another IMAGE_NAME

Reply:
That depends on how the Docker Hub autobuild is configured but since I cant see how the config looks like, this is I had to do it. If we want Docker Hub to build from our Dockerfile directly, this is needed to get the build variables into the image.

Reply:
ok, we can handle that elsewhere. Remove that file.